jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/core/assembly/DockerAssemblyManager.java

```diff
@@ -401,7 +401,7 @@ protected File ensureThatArtifactFileIsSet(JKubeProject project) throws IOExcept
 
     private void setArtifactFile(JKubeProject project, File artifactFile) throws IOException {
         if (artifactFile != null) {
-            File artifact = new File(project.getBuildDirectory() + artifactFile.getName());
+            File artifact = new File(project.getBuildDirectory() + File.separator + artifactFile.getName());
```

Comment:
Try to avoid the use of hard-coded path separators and delegate as much as possible to Java.

Following will achieve the same result
```suggestion
            File artifact = new File(project.getBuildDirectory(), artifactFile.getName());
```